### ISSUE TITLE
are there any preprocessing in video clips ?

### ISSUE THREAD

**Issue by FesianXu** [facebookresearch/VMZ#109]
Hi, great work and gave my lots of help ! However, I still need some help.
I am really not familiar with caffe2 and could not find out whether the caffe2 version irCSN152- IG65M model used any pre-processing to the input video clips or not. (I tried to re-implement the irCSN152 model with pytorch and try to load the pretrained model weights into it.)
In my experiment，I just simply normalized the pixel to [0,1]. but the performance didn't look very good  (about 92% on ucf101, with ig65m pretrained model, I did some finetune on ucf101, or the performance even worse)。 So I wonder if we need to do some specify pre-processing to the video clips like substract the means or somethings else ?
Thanks for your attention and kindly help :)

**Comment by FesianXu** [comment]
find the answer at https://github.com/pytorch/pytorch/blob/master/caffe2/video/video_input_op.h#L374-L378
close the issue now.
